frontends/web/src/routes/account/account.tsx

```diff
@@ -34,6 +34,7 @@ import { Spinner } from '../../components/spinner/Spinner';
 import { Status } from '../../components/status/status';
 import { Transactions } from '../../components/transactions/transactions';
 import { useLoad } from '../../hooks/api';
+import { HideAmountsButton } from '../../components/hideamountsbutton/hideamountsbutton';
```

Comment:
There was a small conflict between `useLoad` and `HideAmountsButton` import, just kept both.